### ISSUE TITLE
[Paper UI] Strange behaviour when trying to select channel title with the mouse

### ISSUE THREAD

**Issue by MHerbst** [eclipse/smarthome#3824]
On the "Things" page I have tried to mark the title of a channel with the mouse. I have created the situation shown in the screenshot by performing the following steps:

- Clicked with the left mouse button on the "V" of Volume
- Moved the mouse with hold left mouse key to the right

Now following happens:
![image](https://user-images.githubusercontent.com/616792/28037817-0cdf1430-65bd-11e7-9043-4500179ec746.png)

All texts were selected (which was not my intention) and a window with a description appeared. The window is nice but wouldn't it be better to have an icon to display this window :-).

Tested with openHAB 2.1 on Windows 10 with Chrome and Edge.

**Comment by htreu** [comment]
I can confirm this behaviour. The popup is shown on "long-press" on the channel label. The selection issue seems to be related: when I temporarily disable the long-press action I can select the text just as expected.

**Comment by htreu** [comment]
If you start selecting the text slightly above the channel label (outside of the actual HTML element) you can select the text w/o forcing the popup and normal text selection behaviour. I would not consider this a bug. But there is certainly room for improvement.

@MHerbst: can we close this then?

**Comment by sjka** [comment]
> The popup is shown on "long-press" on the channel label.

Sounds a bit like an easter-egg... Seriously, I would describe this "feature" rather as _hidden_ or _surprising_, but clearly not intuitive and helpful. Shall be maybe remove it completely?

**Comment by htreu** [comment]
Its actually the only place in PaperUI where the channel-type description can be viewed. I guess we have to provide an alternative way then.
There is already an attempt to update the Thing editor here https://github.com/eclipse/smarthome/pull/3293. Maybe we can include the channel-type description there.

**Comment by kaikreuzer** [comment]
> I would describe this "feature" rather as hidden or surprising

The feature itself was on my request: The description can be very long, so it should not be shown in the normal list, but only upon request. The way it is implemented (seeing a text cursor mouse and only working if pressed on the label) is indeed very hidden/suprising and I only found it myself recently by surprise. The original idea was to follow the Android concepts: A long press on a list item often reveals details or other, not so often used actions on it. So if the new Thing editor could follow that approach, it would imho be best (but I am surely open to other/better suggestions!).

**Comment by sjka** [comment]
Good point, I didn't realize it was the only place where they are shown. So nothing against the feature itself then. 

Still, exposing something only by long-clicking on it is really uncommon on websites. Even when using a browser on an Android phone 😉 

If the new editor would spend a nice, little, not-too-flashy info/details button or a similar visual clue that something is there, that would really help finding it.

**Comment by MHerbst** [comment]
I agree with @sjka that long-clicks are really uncommon on websites respectively on every UI where you are using a mouse.

> If the new editor would spend a nice, little, not-too-flashy info/details button or a similar visual clue that something is there, that would really help finding it

What about adding a clickable image like this 
![image](https://user-images.githubusercontent.com/616792/28082600-309a2f7a-6673-11e7-8c66-a1c47efb68a7.png)
 to signal the availability of additional information. I think it fits quite good to the copy image 
![image](https://user-images.githubusercontent.com/616792/28082667-66b275fe-6673-11e7-87da-ba7356b3ccb7.png)

> @MHerbst: can we close this then?

I agree with you that the "selection problem" is only a minor problem. But I think we need a visual hint that there is additional information instead (or in addition) of using a "long press". I could change the abstract or create a new issue if you want.

**Comment by kaikreuzer** [comment]
> What about ... 

Again, instead of inventing our own UX, I would suggest to check what is the suggested approach according to Material Design UX - so as @htreu tries to follow this with the new list/detail/editor, I am sure he will come up with a good solution :-)
